apps/site/lib/site_web/controllers/customer_support_controller.ex

```diff
@@ -249,6 +251,18 @@ defmodule SiteWeb.CustomerSupportController do
   defp validate_privacy(%{"privacy" => "on"}), do: :ok
   defp validate_privacy(_), do: "privacy"
 
+  @spec validate_vehicle(map) :: :ok | String.t()
+  defp validate_vehicle(%{"vehicle" => ""}), do: :ok
+
+  defp validate_vehicle(%{"vehicle" => vehicle_number}) do
+    case Regex.run(~r/^[0-9]+$/, vehicle_number) do
```

Comment:
- You can use the `*` regex here as well.
- You can also use `Regex.match?` since you don't care about the results.